source/frontend/src/features/mapSideBar/shared/FileMenuView.tsx

```diff
@@ -53,44 +96,56 @@ const FileMenuView: React.FunctionComponent<React.PropsWithChildren<IFileMenuPro
         </Col>
       </StyledRow>
       <StyledMenuHeaderWrapper>
-        <StyledMenuHeader>Properties</StyledMenuHeader>
-        <RestrictedEditControl
-          canEdit={canEdit}
-          isInNonEditableState={isInNonEditableState}
-          icon={<EditPropertiesIcon />}
-          title="Change properties"
-          toolTipId={`${file?.id ?? 0}-summary-cannot-edit-tooltip`}
-          editRestrictionMessage={editRestrictionMessage}
-          onEdit={onEditProperties}
-        />
+        <Row noGutters className="w-100">
+          <Col>
+            <StyledMenuHeader>Properties</StyledMenuHeader>
+          </Col>
+          <Col xs="auto">
+            <RestrictedEditControl
+              canEdit={canEdit}
+              isInNonEditableState={isInNonEditableState}
+              icon={<EditPropertiesIcon />}
+              title="Change properties"
+              toolTipId={`${file?.id ?? 0}-summary-cannot-edit-tooltip`}
+              editRestrictionMessage={editRestrictionMessage}
+              onEdit={onEditProperties}
+            />
+          </Col>
+          <Col xs="auto">
+            <LinkButton title="Fit boundaries button" onClick={fitBoundaries}>
+              <PiCornersOut size={18} className="mr-2" />
+            </LinkButton>
+          </Col>
+        </Row>
       </StyledMenuHeaderWrapper>
       <div className={'p-1'} />
       <StyledMenuBodyWrapper>
         {sortedProperties.map((property: ApiGen_Concepts_FileProperty, index: number) => {
           const propertyName = getFilePropertyName(property);
+          const isCurrentProperty = currentPropertyIndex === index;
           return (
-            <StyledRow
-              key={`menu-item-row-${index}`}
-              data-testid={`menu-item-row-${index}`}
-              className={cx('no-gutters', { selected: currentPropertyIndex === index })}
-              onClick={() => {
-                if (currentPropertyIndex !== index) {
-                  onSelectProperty(property.id);
-                }
-              }}
-            >
-              <Col xs="1">{currentPropertyIndex === index && <FaCaretRight />}</Col>
+            <StyledRow key={`menu-item-row-${index}`} className="no-gutters">
+              <Col xs="1">{isCurrentProperty && <FaCaretRight />}</Col>
               <Col xs="auto" className="pr-2">
-                <StyledIconWrapper className={cx({ selected: currentPropertyIndex === index })}>
+                <StyledIconWrapper className={cx({ selected: isCurrentProperty })}>
                   {index + 1}
                 </StyledIconWrapper>
               </Col>
-              <Col>
-                {currentPropertyIndex === index ? (
+              <StyledCol
+                onClick={() => onPropertyClick(index, property.id)}
```

Comment:
nit: why not move the onclick handler to the LinkButton below? Feels odd to have an onclick handler that doesn't fire when the span is rendered - `isCurrentProperty === true`

Reply:
This is to match the previous way the tests were selecting for the class and the testid. I can change it needed.